pkg/test/syncset/syncset.go

```diff
@@ -127,3 +127,9 @@ func WithPatches(patches ...hivev1.SyncObjectPatch) Option {
 		syncSet.Spec.Patches = patches
 	}
 }
+
+func WithResourceParametersEnabled() Option {
```

Comment:
I think I _was_ going to use it to build up the syncset for the UT, but ended up going a different direction. However, pursuant to your other comments below, it may come back into play if I can manage to cut that test over to the existing framework.

Reply:
Now used (with a slight tweak).